app/views/static_pages/press.html.haml

```diff
@@ -3,3 +3,45 @@
 %p
   If you are a member of the press, please contact our press representative at
   = "#{mail_to PRESS_EMAIL, PRESS_EMAIL}.".html_safe
+
+%h3 Double Union in the News
+
+.press
+  #{link_to "Why Silicon Valley Needs the Coder Grrrls of Double Union, the Feminist Hacker Space", "http://www.fastcompany.com/3031944/most-creative-people/why-silicon-valley-needs-the-coder-grrrls-of-double-union-the-feminist-"}
+  .attribution
+    Fast Company
+
+.press
+  #{link_to "The New Disruptors: Doubling Down with Amelia Greenhall", "http://boingboing.net/2014/01/02/new-disruptors-56-double-union.html"}
+  .attribution
+    Boing Boing
+
+.press
+  #{link_to "Women Techies Build A Room of Their Own", "http://missionlocal.org/2013/10/women-techies-build-a-room-of-their-own/"}
+  .attribution
+    Mission Local
+
+.press
+  #{link_to "The Rise of Feminist Hackerspaces and How To Make Your Own", "https://modelviewculture.com/pieces/the-rise-of-feminist-hackerspaces-and-how-to-make-your-own"}
+  .attribution
+    Model View Culture
+
+
+
+/ Why Silicon Valley Needs the Coder Grrrls of Double Union, the Feminist Hacker Space
```

Comment:
We probably don't need these commented out links, but I can remove them in the merge.